src/domains/point_set.jl

```diff
@@ -3,13 +3,11 @@
 # ------------------------------------------------------------------
 
 """
-    PointSet(coords)
-
-A set of points with coordinate matrix `coords`. The number of rows
-of the matrix is the dimensionality of the domain whereas the number
-of columns is the number of points in the set. Alternatively, `coords`
-can be a vector of tuples (i.e. points).
+PointSet(coords)
```

Comment:
Can you please format the docstring as before? 4 whitespaces before the `PointSet` title, and backquotes for the `coords`. Also, an extra space before the header `## Examples`.